firmware/boron-ins-fsm/test/consoleFunctionTests.cpp

```diff
@@ -109,25 +108,25 @@ SCENARIO( "Turn_Debugging_Publishes_On_Off", "[toggle debug flag]" ) {
 SCENARIO( "Change_IM21_Door_ID", "[change door id]" ) {
 
     GIVEN( "A default global door id" ) {
-        globalDoorID.byte1 = (uint8_t)strtol("AA",NULL,16);
-        globalDoorID.byte1 = (uint8_t)strtol("AA",NULL,16);
-        globalDoorID.byte1 = (uint8_t)strtol("AA",NULL,16);
 
-        WHEN( "the function is called with e" ) {
+        WHEN( "The function is called with e and a valid door ID was previously set" ) {
+            im21_door_id_set("56,34,12");
             int returnVal = im21_door_id_set("e");
 
-            THEN( "should return 1" ) {
-                REQUIRE( returnVal == 1);
+            THEN( "The function should return the door ID that was previously set" ) {
+                // There exists a bug where the last two bytes of returnVal are set
+                // to 00 inside this THEN macro, even if the value was previously
+                // correct. 
+                REQUIRE( returnVal == 1193046 );
```

Comment:
I was expecting this to return `5649426` which is 0x563412.

Reply:
After the fix in the previous comment, I also reversed the door ID that was being set to 12,34,56, making this the correct value to check for